common/types/ethasset.go

```diff
@@ -13,10 +13,30 @@ import (
 // EthAsset represents an Ethereum asset (ETH or a token address)
 type EthAsset ethcommon.Address
 
+// NewEthAsset converts an unquoted string to the EthAsset type
```

Comment:
a json string? or eth address?

Reply:
I removed the method. I was using it outside of tests in an earlier version of the PR.